docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.profile

```diff
@@ -192,3 +192,50 @@ function hs_field_helpers_eck_entity_type_insert(EntityInterface $entity) {
     "edit any $eck_type entities",
   ]);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function su_humsci_profile_preprocess_menu(&$variables) {
+  if ($variables['menu_name'] != 'shortcut_menu') {
+    return;
+  }
+
+  $current_user = \Drupal::currentUser();
+  _su_humci_profile_clean_shortcut_links($variables['items'], $current_user);
```

Comment:
Is the "_" private function still a standard in D8? I can't seem to find anything on this officially: https://www.drupal.org/docs/develop/standards/coding-standards

Reply:
hmm not sure. I thought it was always just convention but not really a standard